src/defund.ts

```diff
@@ -37,13 +37,35 @@ function networkToChainId(network: string): number {
     }
 }
 
-function rpcProvider() {
-    return new providers.JsonRpcProvider("your rpc url");
+function rpcProvider(network: string) {
```

Comment:
you can just use `new providers.InfuraProvider(ChainId, InfuraKey);` for simple

Reply:
I will rewrite this for more customization